content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,133 @@
+
+Finally, let’s consider allocation since the graphs above have only covered usage. Every sidecar resource has a default request of 100 millicores vCPU and 128 MB memory. For simplicity, we assume ztunnels and waypoint proxies have similar requests and limits as their sidecar counterparts. Breaking it down, that’s a 98% reduction in allocated resources with ztunnel, and 90% when waypoints are included.
+
+{{< image width="100%"
+    link="allocation.png"
+    caption="CPU and memory allocation and cost breakdown for sidecar and ambient scenarios"
+    >}}
+
+Going further, we can calculate a dollar amount for these numbers by referring to [GCP monthly pricing](https://cloud.google.com/compute/vm-instance-pricing) for a monthly cost per GB memory and per CPU. Consider these two different machines and their costs:
+
+- `n2-standard-4` (4 CPU, 16 GB) at $141/month
+- `n2-highmem-4` (4 CPU, 32 GB) at $191/month
+
+Calculating the difference in memory shows 1GB costs $3.33/month. In our test this brings memory costs from $60 down to only $1 in the ztunnel-only scenario, and $6 with waypoints deployed.  Similarly for CPU:
+
+- `n2-standard-4` (4 CPU, 16 GB) at $141/month
+- `n2-highmem-2` (2 CPU, 16 GB) at $95/month
+
+Calculating the difference in CPU shows 1 CPU costs $23/month. In our test this brings CPU costs from over $300 down to only $7 in the ztunnel-only scenario, and $34 with waypoints deployed.
+
+We can literally put a dollar amount on the savings ambient brings - and so can you!
+
+## Test It Out
+
+In comparison to what most users run Istio with in production, this test cluster is tiny. However, we expect even more savings with larger clusters and when more services are deployed. We encourage everyone to see what the savings with Istio ambient mesh looks like in their environments. Also note that these scripts have been pushed to GitHub so feel free to check them out [here](https://github.com/solo-io/ambient-performance/tree/boutique-demo). For tracking CPU and memory usage throughout the test scenarios, versions of [Prometheus](https://prometheus.io/), [node-exporter](https://prometheus.io/docs/guides/node-exporter/), and [Grafana](https://grafana.com/) are installed. A custom Grafana dashboard was created for observing relevant data, which can be found and imported from GitHub [here](https://github.com/solo-io/ambient-performance/blob/boutique-demo/dashboard/ambient-performance-analysis.json).
+
```

Comment:
```suggestion
For this example, we have used a small test cluster.  However, we expect the savings to scale to larger clusters and those with more services deployed. We encourage you to see what the savings with Istio ambient mesh would be in your own environments, and will be doing performance tests at large scale as we move towards production readiness.

The scripts used here are [available on GitHub](https://github.com/solo-io/ambient-performance/tree/boutique-demo). For tracking CPU and memory usage throughout the test scenarios, [Prometheus](https://prometheus.io/), [node-exporter](https://prometheus.io/docs/guides/node-exporter/), and [Grafana](https://grafana.com/) are installed. [A custom Grafana dashboard](https://github.com/solo-io/ambient-performance/blob/boutique-demo/dashboard/ambient-performance-analysis.json) was created for observing relevant data.
```
scale linearly?

I maintain you would have a far more powerful story if you just took a day or two to run at actual "real production customer" scale.